### ISSUE TITLE
Command Palette and Profile icon menu items requires alt text

### ISSUE THREAD

**Issue by rrhyne** [sourcegraph/sourcegraph#12953]
In the navigation, both the Campaigns icon and the comamnd palette menu icon 

![image](https://user-images.githubusercontent.com/539268/89912872-e9fe4d00-dba7-11ea-8276-19bf1436dde2.png)

and the CloseIcon in dismissable alerts all fail accessibility due to 'Buttons do not have an accessible name'.

**Comment by jasongornall** [comment]
Gone through several rebrand since then. Feel free to reopen if you think this is still an issue
